containers/powershell/.devcontainer/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
 #-------------------------------------------------------------------------------------------------------------
 
-FROM microsoft/powershell
+FROM mcr.microsoft.com/powershell
 
 # Install git, process tools
 RUN apt-get update && apt-get -y install git procps
```

Comment:
One comment, If you cleanup in separate layers, like this, the final image will be clean, but the data will still be in the layer. It's better to cleanup in the same layer.

Reply:
@Tylerleonhardt I created a PR for your repo to fix this.

https://github.com/TylerLeonhardt/vscode-dev-containers/pull/1